oz/Guest.py

```diff
@@ -148,6 +148,9 @@ def __init__(self, tdl, config, auto, output_disk, nicmodel, clockoffset,
         if macaddress is None:
             self.macaddr = oz.ozutil.generate_macaddress()
 
+        # configuration from 'custom' section
+        self.useuefi = oz.ozutil.config_get_boolean_key(config, 'custom', 'useuefi', useuefi)
```

Comment:
Maybe also the `oz.cfg` file should be adjusted with the new `custom` config section.

Reply:
Feel free to take over this, we are not using oz anymore so I will abandon this.